src/content/pages/earn/token-holder.md

```diff
@@ -13,8 +13,8 @@ tokenHolderInfo:
     next for Threshold. Legacy NU or KEEP tokens should first be upgraded to T.
   image: /images/token-holder.png
   buttons:
-    - label: Upgrade to T
-      url: https://dashboard.threshold.network/upgrade
+    - label: Delegate Vote
+      url: https://delegates.threshold.network
```

Comment:
This domain doesn't appear to resolve?

Reply:
asked for clarification on eta here: https://discord.com/channels/866378471868727316/870386636507185202/1178733196263825499